mobly/controllers/android_device_lib/services/snippet_management_service.py

```diff
@@ -49,11 +55,19 @@ def _is_using_client_v2(self):
 
     NOTE: This is a transient function when we are migrating the snippet client
     from v1 to v2. It will be removed after the migration is completed.
+
+    Returns:
+      A bool for whether this service is using snippet client V2.
     """
     if self._use_client_v2_switch is None:
       device_dimensions = getattr(self._device, 'dimensions', {})
-      self._use_client_v2_switch = (device_dimensions.get(
-          'use_mobly_snippet_client_v2', 'false').lower() == 'true')
+      switch_from_dimension = (device_dimensions.get(_CLIENT_V2_CONFIG_KEY,
+                                                     'false').lower() == 'true')
+
+      switch_from_property = (getattr(self._device, _CLIENT_V2_CONFIG_KEY,
```

Comment:
you are getting from "attributes", not really "property"?

Reply:
Modified. Yes, "property" should be a concept when related to testbed config.